<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>536</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="turnonBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border-image: url(:/button/turnon.png);
}
QPushButton:pressed
{
border-image: url(:/button/turnonhover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/temperture.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="speedBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/speedBtn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="modeBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/modeBtn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="IDBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/IDBtn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="turnonBtn_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button/turnoff.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	
	border-image: url(:/button/menunprmal.png);
}
QToolButton:hover
{
	
	border-image: url(:/button/menuhover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
   </widget>
   <widget class="QLabel" name="roomtem">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/roomtem.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="temup">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button/temup.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="temdown">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button/temdown.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/time.png);
font: 12pt &quot;Lato&quot;;
color: rgb(0, 161, 133);</string>
    </property>
    <property name="text">
     <string>time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="blue">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/blue.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="orange">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/orange.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pink">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pink.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mode_cold">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/status/baseline_ac_unit_white_18dp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mode_warm">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/status/baseline_brightness_5_white_18dp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="speed_0">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/status/baseline_looks_white_18dp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="speed_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/status/baseline_looks_one_white_18dp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="speed_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/status/baseline_looks_two_white_18dp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="speed_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/status/baseline_looks_3_white_18dp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tem_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 1075 57pt &quot;Lato&quot;;
color: rgb(0, 161, 133);</string>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="tem_room_label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 1075 65pt &quot;Lato&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/error.png);
font: 8pt &quot;Lato&quot;;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/info.png);
font: 8pt &quot;Lato&quot;;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="warning_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/warning.png);
font: 8pt &quot;Lato&quot;;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="success_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/success.png);
font: 8pt &quot;Lato&quot;;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="unuse">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/white_wendu.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ID_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/menu-2.png);
font: 8pt &quot;Lato&quot;;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>id</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="realspeed">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/status/realspeed2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>541</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/kongtiao.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="waitlabel1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>640</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="graylabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/gray.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="help">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>set temperature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>speed button</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>mode button</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>change temperature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>real speed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>room temperature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>energy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>fee</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_9">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>help/exit help</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>info bar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="help_11">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/help.png);
color: rgb(0, 161, 133);
font: 9pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>ID bar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>graylabel</zorder>
   <zorder>logo</zorder>
   <zorder>turnonBtn_2</zorder>
   <zorder>turnonBtn</zorder>
   <zorder>label</zorder>
   <zorder>speedBtn</zorder>
   <zorder>modeBtn</zorder>
   <zorder>IDBtn</zorder>
   <zorder>toolButton</zorder>
   <zorder>roomtem</zorder>
   <zorder>temup</zorder>
   <zorder>temdown</zorder>
   <zorder>time</zorder>
   <zorder>blue</zorder>
   <zorder>orange</zorder>
   <zorder>pink</zorder>
   <zorder>mode_cold</zorder>
   <zorder>mode_warm</zorder>
   <zorder>speed_0</zorder>
   <zorder>speed_1</zorder>
   <zorder>speed_2</zorder>
   <zorder>speed_3</zorder>
   <zorder>tem_label</zorder>
   <zorder>tem_room_label</zorder>
   <zorder>error_label</zorder>
   <zorder>info_label</zorder>
   <zorder>warning_label</zorder>
   <zorder>success_label</zorder>
   <zorder>unuse</zorder>
   <zorder>ID_label</zorder>
   <zorder>realspeed</zorder>
   <zorder>waitlabel1</zorder>
   <zorder>help</zorder>
   <zorder>help_2</zorder>
   <zorder>help_3</zorder>
   <zorder>help_4</zorder>
   <zorder>help_5</zorder>
   <zorder>help_6</zorder>
   <zorder>help_7</zorder>
   <zorder>help_8</zorder>
   <zorder>help_9</zorder>
   <zorder>help_10</zorder>
   <zorder>help_11</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
